index.md

```diff
@@ -11,7 +11,7 @@ It should be used in conjunction with the
 [DfE Digital Technical Guidance](https://technical-guidance.education.gov.uk),
 when building digital services.
 
-## Profession
+## Professions
```

Comment:
Not sure if this is typo? Should be profession, singular.
And the link to the newly named profession page needs updating